Default (OSX).sublime-keymap

```diff
@@ -0,0 +1,6 @@
+[
```

Comment:
Correct me if I'm wrong, but is the process of adding support for Linux just adding one of these keymap files for Linux, like this one is for OSX?

Reply:
You are correct. But currently, I don't have access to a Linux system. So, I cannot verify if it works. However, I am certain that apart from creating another keymap file, there is nothing else that needs to be done in order to get this plugin working on a Linux system.

Reply:
Alright, can you make a Linux file then, and I'll test it on my machine? I have a Sublime Text 3 license :)

Reply:
I added sublime-keymap for Linux in c161cd6.

Reply:
You can have a look at https://github.com/ankitshubham97/google-java-format-sublime/blob/master/README.md#linuxubuntu-platform to know how to manually install the plugin on Linux.